Docs/ros_launchs.md

```diff
@@ -3,17 +3,15 @@
 ---
 ## carla_ackermann_control.launch
 
-Creates a node to manage a vehicle using Ackermann controls instead of the CARLA control messages. The node reads the vehicle info from CARLA and uses it to define the controller. A simple Python PID is used to adjust acceleration/velocity. It can be installed using: 
+Creates a node to manage a vehicle using Ackermann controls instead of the CARLA control messages. The node reads the vehicle info from CARLA and uses it to define the controller. A simple Python PID is used to adjust acceleration/velocity. 
 ```sh
 pip install --user simple-pid
 ```
-Initial parameters can be set in: `share/carla_ackermann_control/config/settings.yaml`.  
-It is possible to modify the parameters during runtime via ROS dynamic reconfigure.  
+Initial parameters can be set in `share/carla_ackermann_control/config/settings.yaml`. It is possible to modify the parameters in runtime via ROS dynamic reconfigure.  
```

Comment:
At "Initial parameters can be set in `share/carla_ackermann_control/config/settings.yaml`."
The path should be something like

If you installed ROS-Bridge using APT repository '/opt/carla-ros-bridge/melodic/share/carla_ackermann_control/config/settings.yaml'

or If you installed using source repository '<carla-ros-bridge PATH>/catkin_ws/src/ros-bridge/carla_ackermann_control/config/settings.yaml'

Reply:
Updated!